blmath/geometry/apex.py

```diff
@@ -38,7 +38,7 @@ def inflection_points(points, axis, span):
 
     inflection_point_indices = [i for i, b in enumerate(is_inflection_point) if b]
 
-    if len(inflection_point_indices) == 0:
+    if len(inflection_point_indices) == 0: # pylint: disable=len-as-condition
```

Comment:
I'm being a conservative about fixing these due to the absence of tests in this repo. I would be happy to fix in a next pass.